server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/DatabaseChangeRuntimeFlowableTask.java

```diff
@@ -208,20 +210,27 @@ private void modifyTimeoutIfTimeConsumingSqlExists(DelegateExecution execution,
             return;
         }
         Long taskId = FlowTaskUtil.getTaskId(execution);
-        Long preCheckTaskId = FlowTaskUtil.getPreCheckTaskId(execution);
-        TaskEntity preCheckTask = taskService.detail(preCheckTaskId);
-        PreCheckTaskResult preCheckResult =
-                JsonUtils.fromJson(preCheckTask.getResultJson(), PreCheckTaskResult.class);
-        Validate.notNull(preCheckResult, "Pre check task result can not be null");
-        long autoModifiedTimeout = flowTaskProperties.getIndexChangeMaxTimeoutMillisecond();
-        if (Objects.nonNull(preCheckResult.getSqlCheckResult())
-                && preCheckResult.getSqlCheckResult().isTimeConsumingSqlExists()
-                && autoModifiedTimeout > parameters.getTimeoutMillis()) {
-            this.autoModifyTimeout = true;
-            parameters.setTimeoutMillis(autoModifiedTimeout);
-            TaskEntity databaseChangeTaskEntity = taskService.detail(taskId);
-            databaseChangeTaskEntity.setParametersJson(JsonUtils.toJson(parameters));
-            taskService.updateParametersJson(databaseChangeTaskEntity);
+        Optional<Long> preCheckTaskId;
+        try {
```

Comment:
we can set this parameter to true when create a database change task in personal space instead of by try catch: `parameters.isModifyTimeoutIfTimeConsumingSqlExists()`